README.md

```diff
@@ -1,68 +1,130 @@
+# Code (and Model) Repositories
+## GitHub and GitLab
+The Legend platform and project makes use of both GitHub and GitLab.
+### GitHub
+* GitHub, specifically the [FINOS GitHub organization](https://github.com/finos/), is the distributed source and version control management system used for Legend code.
+  * The Legend GitHub repos, like all FINOS repos, are configured to make use of the FINOS [Open Developer Platform](https://odp.finos.org/) tooling.
+* GitHub infrastructure is what's used to do Legend builds (e.g., of [Legend JARS, which are then distributed through Maven Central](https://search.maven.org/search?q=g:org.finos.legend))
+### GitLab
+* GitLab is used for source control of models "under the hood" within the Legend SDLC module. As such the models, written in the Legend language, are stored in GitLab repositories.
 
-# Documentation Strategy
 
-## [Alloy Documentation Site (alloy.finos.org)](https://alloy.finos.org/)
-The [documentation site](https://alloy.finos.org) is built using Docusaurus based on the markdown files in the [`docs`](/docs/) folder. 
+## Repositores
+### Code Repositories (GitHub)
+[This repository (finos/legend)](www.github.com/finos/legend) is the parent repository for the project. It is used principally for overall Legend documentation and issue tracking. The code itself for the 5 modules open sourced thusfar can be found in the following 5 child repositories:
+* [Shared Module](https://github.com/finos/legend-shared): https://github.com/finos/legend-shared
+* [SDLC Module](https://github.com/finos/legend-sdlc): https://github.com/finos/legend-sdlc
+* [Studio Module](https://github.com/finos/legend-studio): https://github.com/finos/legend-studio
+* [Engine Module](https://github.com/finos/legend-engine): https://github.com/finos/legend-engine
+* [Legend Language](https://github.com/finos/legend-pure): https://github.com/finos/legend-pure
+#### Module Specific Issues
+Module specific issues are opened, maintained, and prioritized within the modules specific issue list (e.g., [https://github.com/finos/legend-studio/issues](https://github.com/finos/legend-studio/issues) for Legend Studio). Issues can be aggregated from multiple reposistories as needed through [FINOS organization level GitHub kanban boards("Projects")](https://github.com/orgs/finos/projects).
 
-## [This Repository - GitHub.com (github.com/finos/alloy)](https://github.com/finos/legend)
+### Model Repositories (GitLab)
+Legend models developed on [the external shared instance](https://legend.finos.org/studio), including those built during the Phase 1 pilot described below, are themselves open sourced code and available in the following GitLab repositories:
+* FX Options extensions to the CDM: https://gitlab.legend.finos.org/alloy-pilot/cdm
```

Comment:
Url should be `https://gitlab.legend.finos.org/legend-pilot/cdm` instead of `https://gitlab.legend.finos.org/alloy-pilot/cdm`

Reply:
Fixed @maoo in https://github.com/brooklynrob/alloy/commit/1b82b714254650229abd309d585a1acbe8a56d34